src/svg/qsvggenerator.cpp

```diff
@@ -159,6 +170,10 @@ Q_GUI_EXPORT QImage qt_imageForBrush(int brushStyle, bool invert);
 class QSvgPaintEngine : public QPaintEngine
 {
     Q_DECLARE_PRIVATE(QSvgPaintEngine)
+
+private:
+    int clip_counter = 0;
+    std::map<std::string, int> clip_path_to_id;
```

Comment:
haven't we talked about it couple of days ago and came to conclusion that this should map QString to int?